.github/workflows/flowzone.yml

```diff
@@ -300,37 +300,25 @@ jobs:
         working-directory: .
         shell: bash
 
-    outputs:
-      versionbot: ${{ steps.versionbot_checkout.outcome }}
-
     steps:
-      - name: Checkout versioned source
-        id: versionbot_checkout
-        continue-on-error: true
-        uses: actions/checkout@v3
-        with:
-          ref: versionbot/pr/${{ github.event.pull_request.number }}
-          fetch-depth: 2
-          token: ${{ secrets.FLOWZONE_TOKEN }}
-
-      - name: Assert that versioned source is up-to-date
-        if: ${{ steps.versionbot_checkout.outcome == 'success' }}
-        run: git merge-base --is-ancestor ${{ github.event.pull_request.head.sha }} HEAD
-
-      # Fallback to unversioned source if versionbot branch does not exist
       - name: Checkout original source
-        if: ${{ steps.versionbot_checkout.outcome == 'failure' }}
         uses: actions/checkout@v3
         with:
-          fetch-depth: 1
+          fetch-depth: 0
           token: ${{ secrets.FLOWZONE_TOKEN }}
 
-      # Run balena-versionist in Github actions when the versionbot branch could not be checked out
       - name: Version original source
-        if: ${{ steps.versionbot_checkout.outcome == 'failure' }}
         run: |
-          npm install -g balena-versionist versionist
-          balena-versionist
+          if [ -f repo.yml ]
+          then
+            npm install -g balena-versionist versionist
+            balena-versionist
+            git status
+            VERSION="$(head -n1 VERSION || jq -r '.version' package.json)"
+            git add --all
+            git -c "user.name=Versionist" -c "user.email=versionist@balena.io" commit --allow-empty -m "${VERSION}"
```

Comment:
We also need to update all the places where we check if author contains 'versionbot'

Reply:
I updated the script to use the name `versionbot` instead.